pkg/controllers/podgroup_test.go

```diff
@@ -109,14 +109,13 @@ func Test_Run(t *testing.T) {
 			podNextPhase:      v1.PodSucceeded,
 		},
 		{
-			name:              "Group status convert from scheduling to succeed",
+			name:              "Group status convert from pending to scheduling",
```

Comment:
@czybjtu 
Can you explain why we need to change this test case here?

Reply:
This is a duplicate of line 102.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/2ea5a971705f0ba21052be7edff0f76323190ed7/pkg/controllers/podgroup_test.go#L102